### DISCUSSION TITLE
<Link> does not work

### DISCUSSION THREAD

**Discussion by Roguey-84** [remix-run/react-router#9411] [Q&A]
I've downloaded the latest version of 'react-router-dom': ^6.4.2, and my App doesn't work. I see a blank page. But if I comment out the <Link>, everything works just fine. Does someone have the same behavior? Or did I do something wrong? Here is my code:

```
import React from 'react';
import {Link} from 'react-router-dom';

const Header = () => {
	return (
		<header className='header' id='header'>
                         <nav className='nav container'>
				<Link to='/' className='nav__logo'>HomePage</Link>

			</nav>
		</header>
	)
}

export default Header
```

Can someone help me, please?

**Comment by ryanflorence** [comment]
Not enough code there to know. Have you done the tutorial? https://reactrouter.com/en/main/start/tutorial
